<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="VSI-LAP-45" name="Test" tests="1" failures="0" timestamp="18 Feb 2017 08:45:10 GMT" time="301.658" errors="0">
  <testcase name="@AfterMethod afterMethod" time="0.0" classname="keyword.RunScripts">
    <failure type="com.microsoft.tfs.core.clients.workitem.exceptions.WorkItemException" message="You do not have permission to update this work item.">
      <![CDATA[com.microsoft.tfs.core.clients.workitem.exceptions.WorkItemException: You do not have permission to update this work item.
	at com.microsoft.tfs.core.exceptions.mappers.WorkItemExceptionMapper.mapSoapFaultDetailsID(WorkItemExceptionMapper.java:140)
	at com.microsoft.tfs.core.exceptions.mappers.WorkItemExceptionMapper.mapSoapFault(WorkItemExceptionMapper.java:85)
	at com.microsoft.tfs.core.exceptions.mappers.WorkItemExceptionMapper.map(WorkItemExceptionMapper.java:46)
	at com.microsoft.tfs.core.clients.workitem.internal.update.BaseUpdatePackage.update(BaseUpdatePackage.java:102)
	at com.microsoft.tfs.core.clients.workitem.internal.WorkItemImpl.internalSave(WorkItemImpl.java:673)
	at com.microsoft.tfs.core.clients.workitem.internal.WorkItemImpl.save(WorkItemImpl.java:388)
	at tFS.Defectcreation.createBugsinTFS(Defectcreation.java:88)
	at wrappers.iTF.afterMethod(iTF.java:75)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase name="@AfterTest afterTest" time="0.0" classname="keyword.RunScripts">
    <skipped/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase name="runScripts" time="65.145" classname="keyword.RunScripts"/>
</testsuite> <!-- Test -->
